auth/src/main/java/com/firebase/ui/auth/ErrorCodes.java

```diff
@@ -17,4 +13,8 @@ private ErrorCodes() {
      * An unknown error has occurred
      **/
     public static final int UNKNOWN_ERROR = 20;
+
+    private ErrorCodes() {
```

Comment:
We don't, this was just formalizing the class as a "`Constants`" class that shouldn't be instantiated.